students/tammyd/lesson03/mailroom_pt1.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
+
+"""
+
+Mailroom program for a non profit
+
+"""
+
+import sys
+
+donor_db = [("Britney Spears", [50.99, 100.99, 2000.99]),
+            ("Christina Aguilera", [50.99, 100.99, 2000.99]),
+            ("Mandy Moore", [50.99, 100.99, 2000.99]),
+            ("Jessica Simpson", [50.99, 100.99, 2000.99]),
+            ("Queen Bey", [50.99, 100.99, 2000.99]),
+            ]
+
+def mainloop():
+    print("Welcome to the Mailroom \n")
+
+def main_menu():
+    print("Main Menu:")
+    print(
+      "T - Send a thank you\n"
+      "R - Create a report\n"
+      "Q - Quit the program"
+      )
+    menu_selection = input("\nChoose a letter:  ")
```

Comment:
We probably want something like:
```
while True:
```
around our menu, so that  it keeps running without each function needing to call main_menu()